contents/docs/api/trends.mdx

```diff
@@ -1,80 +1,320 @@
 ---
-title: Trends
+title: Insights
 sidebar: Docs
 showTitle: true
 ---
 
 > For instructions on how to authenticate to use this endpoint, see [API Overview](/docs/api/overview).
 
-In PostHog, there is an API endpoint available to do advanced analytics on your data. We use this endpoint for the [Trends page](/docs/features/trends) in PostHog.
+The `/api/insight/` endpoints allow you to fetch insights that provides advanced analytics on the data from PostHog.
 
-## List Trends
+## Endpoints
+<Endpoint endpoint='/api/insight' allowedMethods={['get']} />
+
+Get a list of insights from PostHog.
+
+<HiddenSection headingType='h4' title='Query Parameters'>
 
-```shell
-GET /action/trends/?events=[{"id":"$pageview"}]
-GET /action/trends/?events=[{"id":"$pageview"}]&properties=[{"key":"$browser","value":"Chrome"}]
-```
 
-| Attribute | Type | Required | Description |
-| --- | --- | --- | --------------------------------------------------------------------------------------------------------------------------------------------------- |
-| `events` | Array of Events | yes | Basic usage is `[{"id": "$pageview"}]`.<br /><br />For each event, you can specify:<br />- `id` Name of the event<br />- `properties` A Properties object (see below)<br />- `order` (int) order of the event. This is mostly used for funnels |
-| `properties` | Array of Properties | no | The key/values that you want to filter on. Basic usage is: `[{"key": "$browser", "value": "Chrome"}]`<br /><br />For each property, you can specify:<br />- `key` Key of the property<br />- `value` Value you want to filter on<br />- `type` Either `person` or `event`<br />- `operator`, any of: <br />-- `exact`/empty<br />--`is_not`<br />--`icontains`<br />--`not_icontains`<br />--`gt`<br />--`lt`<br />--`is_set` |
-| `interval` | String | no | Interval of the data. `minute`, `hour`, `day`, `week`, `month`
-| `breakdown_value` | String | no | Split out all entities by any property value or cohort id.  
-| `breakdown_type` | String | no | Default: `property`. Can be set to `cohort`, in which case you can pass through a cohort_id to `breakdown_value`
+| Parameter | Type / Valid Values | Description | Required |
+| --- | --- | --- | --- |
+| `properties` | Array of Properties | The key/values that you want to filter on. Basic usage is: `[{"key": "$browser", "value": "Chrome"}]`<br /><br />For each property, you can specify:<br />- `key` Key of the property<br />- `value` Value you want to filter on<br />- `type` Either `person` or `event`<br />- `operator`, any of: <br />-- `exact`/empty<br />--`is_not`<br />--`icontains`<br />--`not_icontains`<br />--`gt`<br />--`lt`<br />--`is_set` | ❌ |
+| `saved` | boolean |  | ❌ |
```

Comment:
Could you standardize the type naming? I think we should use `string`, `boolean`, and `int`, all lowercase and formatted as code (i.e. using backticks)